.github/workflows/testing.yml

```diff
@@ -26,6 +27,7 @@ jobs:
       with:
         path: wlan-pki-cert-scripts
         repository: Telecominfraproject/wlan-pki-cert-scripts
+        ref: local/dev
```

Comment:
@4c74356b41 I think we can merge this to master as part of this merge?

Reply:
i'd then need to address the comments on that one, maybe its better to do it directly after?

Reply:
@4c74356b41 ok, lets do it after we merge this PR and then fix back.

Reply:
@4c74356b41 now that I think about this... You won't be able to merge this PR without merging the local/dev branch in the PKI script, because otherwise, you will break the cloudsdk deployment for everyone who is using the PKI script with the master.